flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/iceberg/FlinkIcebergRestCatalogIT.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.flink.connector.integration.test.iceberg;
+
+import com.google.common.collect.Maps;
+import java.util.Arrays;
+import java.util.Map;
+import org.apache.flink.table.api.ResultKind;
+import org.apache.flink.types.Row;
+import org.apache.gravitino.Schema;
+import org.apache.gravitino.flink.connector.iceberg.IcebergPropertiesConstants;
+import org.apache.gravitino.flink.connector.integration.test.utils.TestUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIf;
+
+@Tag("gravitino-docker-test")
+public class FlinkIcebergRestCatalogIT extends FlinkIcebergCatalogIT {
+
+  @Override
+  protected Map<String, String> getCatalogConfigs() {
+    Map<String, String> catalogProperties = Maps.newHashMap();
+    catalogProperties.put(
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_BACKEND,
+        IcebergPropertiesConstants.ICEBERG_CATALOG_BACKEND_REST);
+    catalogProperties.put(
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_WAREHOUSE, warehouse);
+    catalogProperties.put(
+        IcebergPropertiesConstants.GRAVITINO_ICEBERG_CATALOG_URI, icebergRestServiceUri);
+    return catalogProperties;
+  }
+
+  @Override
+  public void testListSchema() {
```

Comment:
any reason to add `testAlterSchemaWithCommentAndOptions` and `testListSchema` in REST IT?

Reply:
The following two judgments cannot pass in CI, but they can be successful locally. I haven't found the real reason yet:
1. testListSchema  `Assertions.assertEquals(1, catalog.asSchemas().listSchemas().length);`
3. testAlterSchemaWithCommentAndOptions    `Assertions.assertNotNull(loadedSchema.properties().get("location"));`

Reply:
`Assertions.assertNotNull(loadedSchema.properties().get("location"));` check seems ambiguous, the action may differ by underlying catalog implements, How about remove the check from origin test? 
It's odd for `testListSchema` cases. I will pickup some time to check it in these days too.

Reply:
> `Assertions.assertNotNull(loadedSchema.properties().get("location"));` check seems ambiguous, the action may differ by underlying catalog implements, How about remove the check from origin test? It's odd for `testListSchema` cases. I will pickup some time to check it in these days too.

ok

Reply:
@FANNG1 fixed all,  PTAL again.